apps/journeys-admin/src/components/ContainedIconButton/ContainedIconButton.tsx

```diff
@@ -0,0 +1,75 @@
+import { ReactElement, ReactNode } from 'react'
+import Box from '@mui/material/Box'
+import Typography from '@mui/material/Typography'
+import Card from '@mui/material/Card'
+import CardActionArea from '@mui/material/CardActionArea'
+import Stack from '@mui/material/Stack'
+import Icon from '@mui/material/Icon'
+import { ImageThumbnail } from '../ImageThumbnail'
+
+interface ContainedIconProps {
+  thumbnailIcon: ReactNode
+  actionIcon: ReactNode
+  label: string
+  description?: string
+  imageSrc?: string
+  imageAlt?: string
+  loading?: boolean
+  onClick: () => void
+}
+
+export function ContainedIconButton({
+  thumbnailIcon,
+  actionIcon,
```

Comment:
The ImageThumbnail already handles a default icon if one is not given. Once you do this

- on stories, for the default story, you should no longer need to pass in thumbnailIcon and actionIcon
- on tests, add a test to see if a custom `actionIcon` will show.

```suggestion
  thumbnailIcon,
  actionIcon = <AddIcon />,
```

Reply:
fixed in 1dca4540ebe0a2853e4ff46a2f5f024b0153a5d0